src/quic/logstream.cc

```diff
@@ -28,14 +28,14 @@ Local<FunctionTemplate> LogStream::GetConstructorTemplate(Environment* env) {
     tmpl->InstanceTemplate()->SetInternalFieldCount(
         StreamBase::kInternalFieldCount);
     tmpl->SetClassName(state.logstream_string());
-    StreamBase::AddMethods(env, tmpl);
+    AddMethods(env, tmpl);
```

Comment:
This one I'd prefer to keep for code clarity.

Reply:
Agreed. Removed this change from the PR.